appsflyer/build.gradle

```diff
@@ -34,7 +34,7 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.0.2'
     implementation 'androidx.core:core-ktx:1.0.2'
     api 'com.tealium:remotecommands:1.0.0'
```

Comment:
should probably update this at the same time?
and add the version info into the RemoteCommand(...) constructor which appears to have not been done yet for this RC